README.md

```diff
@@ -16,13 +15,63 @@ And then execute:
 $ bundle
 ```
 
-Or install it yourself as:
-```bash
-$ gem install govuk_publishing_components
+### Mount the component guide
+
+Mount the component guide in your application:
+
+```ruby
+# config/routes.rb
+unless Rails.env.production?
+  mount GovukPublishingComponents::Engine, at: "/component-guide"
+end
+```
+
+For example, if your application was government-frontend, the component guide would be found at:
+`government-frontend.dev.gov.uk/component-guide`
+
+### Write components
+
+`govuk_publishing_components` expects the following components structure in your application:
+
+* a Partial View in `app/views/components` eg `_component-name.html.erb` - The template logic and markup, also defines the arguments expected
+* Documentation in `/app/views/components/docs` - a `.yml` per component, describing the component and containing fixture data.
+* a SCSS module in `app/assets/stylesheets/components` - The styling of the component
+* a Javascript module in `app/assets/javascripts/components` - JS behaviour of the component
+* a unit test in `/test/components` - testing the component renders correctly based on parameters passed in
+
+The component guide will include your application’s styles and scripts. It will include `application.scss` and `application.js` by default but this is [configurable](#configuration).
+
```

Comment:
Should there be a line or two in here about how to call/use a component? I guess it's already going to be in the component guide though.